src/m365/spo/commands/file/file-roleinheritance-reset.ts

```diff
@@ -0,0 +1,145 @@
+import { Cli, Logger } from '../../../../cli';
+import { AxiosRequestConfig } from 'axios';
+import Command from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import * as SpoFileGetCommand from './file-get';
+import { Options as SpoFileGetCommandOptions } from './file-get';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  fileUrl?: string;
+  fileId?: string;
+  confirm?: boolean;
+}
+
+class SpoFileRoleInheritanceResetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.FILE_ROLEINHERITANCE_RESET;
+  }
+
+  public get description(): string {
+    return 'Restores the role inheritance of a file';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        fileUrl: typeof args.options.fileUrl !== 'undefined',
+        fileId: typeof args.options.fileId !== 'undefined',
+        confirm: typeof args.options.confirm !== 'undefined'
```

Comment:
Confirm option shouldn't be anonymized, also if you now pass `debug: false`, the telemetry will log it as true.
We should just convert it to a boolean, even if it is undefined.

```suggestion
        confirm: !!args.options.confirm
```